.circleci/config.yml

```
@@ -97,6 +97,7 @@ jobs:
       LANG: en_US.UTF-8
       LC_CTYPE: en_US.UTF-8
       SHELL: /bin/bash
+      # DOCS_BUILD_REF: {{ .Environment.CIRCLE_SHA1 }}
     working_directory: ~/multicast
```

There is a commented-out line here. If this line isn't needed in version control, consider accompanying it with a comment explaining why it's commented out.

```suggestion
      # Use DOCS_BUILD_REF to set which documents are built, when building documentation with `make build-docs`
      # DOCS_BUILD_REF: {{ .Environment.CIRCLE_SHA1 }}
```

---

There is a commented-out line here. If this line isn't needed in version control, consider accompanying it with a comment explaining why it's commented out.
```suggestion
    working_directory: ~/multicast
    # Use DOCS_BUILD_REF to set which documents are built, when building documentation with `make build-docs`
```